src/web/components/elements/YoutubeBlockComponent.tsx

```diff
@@ -192,7 +192,13 @@ export const YoutubeBlockComponent = ({
                 width={width}
                 title={mediaTitle}
                 duration={duration}
-                origin={origin}
+                origin={
+                    typeof window !== 'undefined' &&
+                    window.location.hostname !== 'localhost' &&
+                    window.location.hostname !== 'm.code.dev-theguardian.com'
+                        ? origin
+                        : ''
+                }
```

Comment:
This code looks like it only gets added to make the test work? Ideally, we keep test implementations out of the live code as it adds a cognitive overload that we really don't want. That is, it will be hard for people in the future to know what this code is for and if they need to worry about it when refactoring / adding features.